dateparser/date_parser.py

```diff
@@ -32,10 +34,65 @@ def __add__(self, other):
 parser.relativedelta.relativedelta = new_relativedelta
 
 
+class new_parser(parser.parser):
+    """
+    Implements an alternate parse method which supports preference to dates in future and past.
+    For more see issue #36
+    """
+
+    def parse(self, timestr, default=None, ignoretz=False, prefer_future=None, **kwargs):
+        # timestr needs to be a buffer as required by _parse
+        timestr = timestr if not isinstance(timestr, str) else StringIO(timestr)
+
+        # Set right prefer_future var
+        prefer_future = prefer_future if prefer_future is not None \
```

Comment:
We expect to have many settings in the future. Let's not use method argument (or it will very long list some day) and only use `settings`.